lib/gems/pending/appliance_console/cli.rb

```diff
@@ -144,6 +154,7 @@ def run
       uninstall_ipa if uninstall_ipa?
       install_ipa if install_ipa?
       install_certs if certs?
+      set_date_time if date_time?
```

Comment:
Ok... I though that I needed date and time for autosync but not...I'll make the change